ngraph/core/include/ngraph/type/element_type.hpp

```diff
@@ -130,21 +131,50 @@ namespace ngraph
 
         typedef std::vector<Type> TypeVector;
 
+        NGRAPH_DEPRECATED(
+            "This global element type was deprecated. Please use Type_t::undefined instead.")
```

Comment:
Just for my curiosity. Why we don't use `constexpr` and use `extern` for variables which might be inline?

Reply:
I will recheck, if it works I will change it and remove deprecation.